cli/content-version.js

```diff
@@ -0,0 +1,53 @@
+const fs = require("fs");
+const path = require("path");
+const { Repository } = require("nodegit");
+
+const PROJECT_ROOT = path.join(__dirname, "..");
+const SHA_PATH = path.join(PROJECT_ROOT, "content.gitsha");
+const CONTENT_NAME = "stumptown";
+
+async function retrieveLocalVersion() {
+  const repository = await Repository.open(
+    path.join(PROJECT_ROOT, CONTENT_NAME)
+  );
+  const commit = await repository.getHeadCommit();
+  return commit.sha();
+}
+
+async function retrieveRemoteVersion() {
+  const repository = await Repository.open(PROJECT_ROOT);
+  const submodule = (await repository.getSubmodules()).find(
+    submodule => submodule.name() === CONTENT_NAME
```

Comment:
...and this would then become a problem if I have `process.env.STUMPTOWN_CONTENT_ROOT` set. 
Actually, I think the business logic should be that if `process.env.STUMPTOWN_CONTENT_ROOT` is truthy and it's different from the default, it should bail entirely on checking `if (remoteVersion !== localVersion) {`.
I.e.
```diff
- if (remoteVersion !== localVersion) {
+ if (!process.env.STUMPTOWN_CONTENT_ROOT && remoteVersion !== localVersion) {
    return VersionStatus.REMOTE_CHANGES;
  }
```

I know both Daniel and Will have been using `STUMPTOWN_CONTENT_ROOT` when they're testing out their bleeding edge versions of `stumptown-content` with the renderer. It's because they're working on something related to `build-json` and then they want to check it out on a local dev server.